armi/utils/tests/test_flags.py

```diff
@@ -152,3 +153,45 @@ def test_hashable(self):
 
     def test_getitem(self):
         self.assertEqual(ExampleFlag["FOO"], ExampleFlag.FOO)
+
+    def test_duplicateFlags(self):
+        """Show that duplicate flags can be added and silently ignored."""
+
+        class F(Flag):
+            @classmethod
+            def len(cls):
+                return len(cls._nameToValue)
+
+        F.extend({"FLAG0": auto()})
+        for i in range(1, 12):
+            F.extend({f"FLAG{i}": auto()})
+            num = F.len()
+            F.extend({f"FLAG{i - 1}": auto()})
+            self.assertEqual(F.len(), num)
+
+            ff = getattr(F, f"FLAG{i}")
+            FlagSerializer._packImpl(
```

Comment:
Can you add a comment here about why these lines are included for future people? They aren't actually used in the test, but they are showing that this can be done without raising an error like it did prior to this change.